app/src/main/java/com/feedhenry/securenativeandroidtemplate/domain/services/MobileCoreService.java

```diff
@@ -0,0 +1,28 @@
+package com.feedhenry.securenativeandroidtemplate.domain.services;
+
+import android.content.Context;
+
+import org.aerogear.mobile.core.MobileCore;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+/**
+ * Created by tjackman on 22/02/18.
+ */
+
+@Singleton
+public class MobileCoreService {
```

Comment:
is this actually required? can we not just return an instance of MobileCore in the `SecureApplicationModule.java` class?

Reply:
@wei-lee Sorry, missed a commit from a few days ago. I have brought it in now. Can you re-check?